lib/semian/unprotected_resource.rb

```diff
@@ -6,6 +6,8 @@ class UnprotectedResource
 
     def initialize(name)
       @name = name
+      @bulkhead = nil
```

Comment:
I believe both of these initializations are unnecessary? Are `@name` and `@bulkhead` `nil` by default?